tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py

```diff
@@ -0,0 +1,226 @@
+# Python imports
```

Comment:
Running pylint locally gave these relevant suggestions.
```

pylint tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py 
************* Module tests.unit_tests.data_steward.retraction.retract_deactivated_pids_test
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:60:12: W1505: Using deprecated method assertEquals() (deprecated-method)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:50:16: W0612: Unused variable 'i' (unused-variable)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:101:16: W0612: Unused variable 'i' (unused-variable)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:226:8: W1505: Using deprecated method assertEquals() (deprecated-method)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:20:0: W0612: Unused variable 'RetractDataBqTest' (unused-variable)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:16:0: C0412: Imports from package constants are not grouped (ungrouped-imports)
tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py:17:0: C0412: Imports from package retraction are not grouped (ungrouped-imports)
```
The deprecated alias [is documented here](https://docs.python.org/3.7/library/unittest.html#deprecated-aliases).  I had to look that one up because I wasn't expecting it.

Reply:
thank you for that link!